S/SimpleITK/build_tarballs.jl

```diff
@@ -0,0 +1,78 @@
+using BinaryBuilder, Pkg
+
+name = "SimpleITK"
+version = v"2.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://github.com/SimpleITK/SimpleITK/releases/download/v$(version)/SimpleITK-$(version).tar.gz", "b07bb98707556ebc2b79aac22dc14950749f509e5b43da8043233275aa55488a")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+mount -t tmpfs -o size=12G tmpfs /workspace/srcdir
+cd ..
+cd ..
+cd workspace/srcdir
+wget https://github.com/SimpleITK/SimpleITK/releases/download/v2.2.0/SimpleITK-2.2.0.tar.gz
+tar -xvzf ./SimpleITK-2.2.0.tar.gz 
+mkdir SimpleITK-build
+cd SimpleITK-build/
+cmake -DCMAKE_INSTALL_PREFIX:FILEPATH=/workspace/destdir -DCMAKE_BUILD_TYPE:STRING=RELEASE -DBUILD_SHARED_LIBS:BOOL=ON ../SimpleITK-2.2.0/SuperBuild
+make -j${nproc}
+rm -rf /workspace/srcdir/SimpleITK-build/ITK-build/lib/cmake
+cd SimpleITK-build
+make install
+cp /workspace/srcdir/SimpleITK-build/ITK-build/lib/* /workspace/destdir/lib
```

Comment:
So basically, the build process of SimpleITK which relies on further low level library backend ITK, produces shared objects for SimpleITK (in the configured prefix path) and the ITK in  a subdirectory, i needed to copy all of the ITK .so files in the SimpleITK prefix path to ensure libraries get linked.

Reply:
That should be in a separate ITK package then.

Reply:
The backend of SImpleITK is ITK, the copying is done just to link stuff. Otherwise it results in a bunch of errors and fails the process of binarybuilder linker.

Reply:
ok so add that jll package as a dependency for this package? also i am going to be building the same thing twice, the ITk build results from the build of SimpleITK